easybuild/easyconfigs/t/torchtext/torchtext-0.10.0-fosscuda-2020b-PyTorch-1.9.0.eb

```diff
@@ -0,0 +1,39 @@
+easyblock = 'PythonPackage'
+
+name = 'torchtext'
+version = '0.10.0'
+local_pytorch_version = '1.9.0'
+local_python_suffix = '-Python-%(pyver)s'
```

Comment:
Looks like this variable/line is redundant, now that the Python version has been dropped from the `versionsuffix`.
```suggestion
```